tests/tester.nim

```diff
@@ -827,3 +826,17 @@ test "init does not overwrite existing files (#581)":
     check execNimbleYes("init").exitCode == QuitSuccess
     check readFile("src/issue581.nim") == Src
   removeDir("issue581")
+
+test "remove skips packages with revDeps (#504)":
+  check execNimble("install", "nimboost", "nimfp", "-y").exitCode == QuitSuccess
```

Comment:
It's great that you've added a test here, but this looks flaky. At least please specify a commit hash so that it's less likely that an installation will fail.

Reply:
I've linked to specific versions.